metaci/build/models.py

```diff
@@ -119,6 +124,9 @@ class Build(models.Model):
     build_type = models.CharField(max_length=16, choices=BUILD_TYPES, default='legacy')
     user = models.ForeignKey('users.User', related_name='builds', null=True, on_delete=models.PROTECT)
 
+    release_relationship_type = models.CharField(max_length=50, choices=RELEASE_REL_TYPES, null=True, blank=True)
```

Comment:
I get the distinction between manual and automated, but what's the distinction between Release Test and Release Automation about?

Reply:
Release Test = a build related to the output of a release - runs all tests on managed package for example

Release Automation = a build related to the creation of a release, once we start automating those